content/swiftui/concepts/widgets/widgets.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'SwiftUI Widgets'
+Description: 'Building user interfaces across Apple platforms.'
+Subjects: 
+    - 'iOS Development'
+Tags: 
+    - 'SwiftUI'
+    - 'User Interface'
+    - 'iOS'
+CatalogContent: 
+    - 'paths/iOS Development'
+    - 'Learn SwiftUI'
+---
+
+# SwiftUI Widgets
+
+## What are SwiftUI Widgets?
+**SwiftUI Widgets** are interactive components that provide users with glanceable information or quick access to app functionality directly from the home screen of their device. They are mini-applications that can display dynamic content, such as weather updates, calendar events, or news headlines. SwiftUI widgets can be created using the WidgetKit framework and are highly customizable, allowing developers to design visually appealing and personalized experiences for users without requiring them to open the full app.
+
+## To create a widget using the Widget Extension template in Xcode, follow these steps:
```

Comment:
```suggestion
## Xcode Widget Extension Template
```

Reply:
This section is getting into the "tutorial realm" which we would like to differentiate Docs from. Rather than set-up, I think it would be more helpful to perhaps explain in this section how Xcode offers developers a template to use which "will generate the necessary files and folders for the widget extension, including a default SwiftUI widget view and a widget extension entry point."

You can also go into other info about this template as you've done here after the steps.